ui/app/mainui/Popups.qml

```diff
@@ -1225,6 +1225,13 @@ QtObject {
         Component {
             id: swapModal
             SwapModal {
+                swapInputParamsForm: SwapInputParamsForm {}
```

Comment:
No need to initialize it to empty/default value if the only way to invoke this is via `openSwapModal(parameters)`